modules/nw-networkpolicy-about.adoc

```diff
@@ -51,7 +51,7 @@ spec:
   - from:
     - namespaceSelector:
         matchLabels:
-          network.openshift.io/policy-group: ingress
+          policy-group.network.openshift.io/ingress:
```

Comment:
@JoeAldinger Change it have no value as below to follow key: value pattern
<pre>
policy-group.network.openshift.io/ingress: ""
</pre>

Reply:
Good catch, thanks Arti